packages/formation-react/src/components/MegaMenu/Column.jsx

```diff
@@ -57,14 +61,15 @@ const Column = props => {
   return (
     <div
       aria-hidden={hidden ? 'true' : 'false'}
-      className={`vetnav-panel vetnav-panel--submenu ${_.kebabCase(
-        keyName,
-      )}${isPanelWhite(mobileMediaQuery, panelWhite)}`}
+      className={`vetnav-panel vetnav-panel--submenu ${columnClassName}${isPanelWhite(
+        mobileMediaQuery,
+        panelWhite,
+      )}`}
     >
-      <h3 id={`vetnav-${_.kebabCase(keyName)}-header`}>{data.title}</h3>
+      <h3 id={`vetnav-${ariaLabelledBy}-header`}>{data.title}</h3>
       <ul
-        id={`vetnav-${_.kebabCase(keyName)}-col`}
-        aria-labelledby={`vetnav-${_.kebabCase(keyName)}-header`}
+        id={`vetnav-${ariaLabelledBy}-col`}
+        aria-labelledby={`vetnav-${ariaLabelledBy}-header`}
```

Comment:
The ` aria-labelledby` has to be unique for the Axe check to pass. With the whole menu generating during startup, we had to use something other than the "columnOne" for that value, since each menu has columns.